services/fluent-bit/0.20.9/resource-quota.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   wait: true
   interval: 6h
   retryInterval: 1m
-  path: ./services/fluent-bit/0.20.5/resource-quota
+  path: ./services/fluent-bit/0.20.9/resource-quota
```

Comment:
@bcmendoza fyi (same deal as my comment on the other pr) this is another one that I manually updated